### ISSUE TITLE
Swipe screen to change location

### ISSUE THREAD

**Issue by OddSobriquet** [davidtakac/prognoza#34]
Current behaviour:
    - swipe to the right to open menu-bar.

Suggested behaviour:
    - Swipe left /right to change displayed location
![IMG_20221222_194401.jpg](https://user-images.githubusercontent.com/72888738/209204845-79551c1b-a979-47e1-a018-26d12c437901.jpg)
    - Swipe from edge into the screen to open menubar![IMG_20221222_194419.jpg](https://user-images.githubusercontent.com/72888738/209204871-9bd197b2-3eac-455f-99e3-b82a6ef2b880.jpg)

Motivation: intuitively I tried to "scroll" / swipe through my save / added locations. Unfortunately, while in other apps a common mechanic, this was not possible.
It would allow for quick and intuitive comparison of two or more different locations.

**Comment by davidtakac** [comment]
Oh you're right, that would be awesome! I'll have to rethink the forecast fetching logic a bit, but I like the idea.

**Comment by OddSobriquet** [comment]
I know the dangers of "take your time" but please, take your time.
This is merely an EOL improvement and nothing too vital. It's still a very well designed app. ☺️

**Comment by davidtakac** [comment]
Thank you, that's very kind 🙂

**Comment by 51k34** [comment]
if i could interject: 
The swipe from edge feature would interfere with the navigation on some android devices. 
Where, to go back, you swipe from the edge. Making it impossible to use the new feature.

I do not know how widespread the feature is, but it is seen as the "modern" way of navigating.

**Comment by 51k34** [comment]
> The swipe from edge feature would interfere with the navigation on some android devices. 


But it's still a good feature. And i hope to see it well implemented at some point.

**Comment by davidtakac** [comment]
Yes, you are right. If this were implemented, you would be able to swipe left/right amywhere on the screen to trigger this behavior. Kind of like a pager. This would not interfere with gesture navigation.

**Comment by OddSobriquet** [comment]
> The swipe from edge feature would interfere with the navigation on some android devices. 

I use swipe navigation. At least on my device (miui12) the "go back" is the bottom half of the screen's edge leaving the top half usable

Edit, it's not exactly split 50:50 it's more of a 1/3:2/3 kinda split with the bottom 2/3 used for the go back.

**Comment by 51k34** [comment]
> on my device (miui12) the "go back" is the bottom half of the screen's edge leaving the top half usable

Oh guess it varies from phone to phone. On mine its the whole of the two edges
